opentimelineio/opentime.py

```diff
@@ -41,6 +41,18 @@ def __init__(self, value=0, rate=1):
         self.value = value
         self.rate = rate
 
+    def __copy__(self):
+        return self.__deepcopy__()
+
+    def __deepcopy__(self, memodict={}):
```

Comment:
Not at all related to my request, but was curious about memodict as I have never used that. However this did trigger my "no-multiple objects as default arguments" concern: 

https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
Yeah, that thought occurred to me too. I want to ask @ssteinbach about it, as that default argument was present in other code already.